src/System.Net.Requests/src/System.Net.Requests.csproj

```diff
@@ -50,8 +47,20 @@
       <Link>Common\System\NotImplemented.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' ">
+    <Compile Include="System\Net\WebExceptionPal.Windows.cs" />
+    <Compile Include="$(CommonPath)\Interop\Windows\wininet\Interop.wininet_errors.cs">
+      <Link>Common\Interop\Windows\wininet\Interop.wininet_errors.cs</Link>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">
+    <Compile Include="$(CommonPath)\Interop\Unix\libcurl\Interop.libcurl_curlcode.cs">
+      <Link>Common\Interop\Unix\libcurl\Interop.libcurl_curlcode.cs</Link>
+    </Compile>
+    <Compile Include="System\Net\WebExceptionPal.Unix.cs" />
```

Comment:
Nit: it'd be nice if the files were in the same order in the Unix section as in the Windows section.